### DISCUSSION TITLE
Supported device types

### DISCUSSION THREAD

**Discussion by tylerjwilliams** [alandtse/alexa_media_player#1155] [Q&A]
Is there a list of supported devices? I recently added a Fire Cube which has an active alexa client which is always active (even in standby).  However, the device doesn't seem to present itself in Alexa Media Player.  Let me know if there is anything I can provide.  Verified that enable newly added entities is activated.

**Comment by alandtse** [comment]
We require the device to show it has certain [capabilities](https://github.com/custom-components/alexa_media_player/blob/dev/custom_components/alexa_media/__init__.py#L444). If it doesn't, we won't add it as a media_player. You can find the device capabilities here: https://alexa.amazon.com/api/devices-v2/device (use the right domain).

If you believe the Fire Cube has capabilities we should include, paste them here.

You can force the device to be added by adding it in the `include_devices` list. Use the `accountName` (case sensitive).
